config/application.rb

```diff
@@ -137,7 +137,11 @@ class Application < Rails::Application
     # DEFAULTS
 
     # Default branding image if the user does not specify one
-    config.branding_image_default = "https://raw.githubusercontent.com/bigbluebutton/greenlight/master/app/assets/images/logo_with_text.png"
+    if ENV["DEFAULT_LOGO"].nil?
```

Comment:
You can replace the whole if statement with `config.branding_image_default = ENV["DEFAULT_LOGO"].presence || "https://raw.githubusercontent.com/bigbluebutton/greenlight/master/app/assets/images/logo_with_text.png"
`

Reply:
That's correct, however, Rubocop doesn't like the length of that particular line, so I split it up into smaller statements.

Reply:
I don't believe rubocop checks the `application.rb`. Tested it locally and rubocop seems to be okay with it